### PR TITLE
Added LITE Libraries and added "AmmoType:" and "UsesAmmoType:" categories to many ranged weapons (in support of Fantasy Grounds export)

### PR DESCRIPTION
**Pull request by crnormand**

### CODE REVIEW

File: Library/Equipment/Dungeon Fantasy RPG.eqp [hunk 1]
```diff
@@ -10914,17 +10932,7 @@
...
-	<equipment version="4">
-		<quantity>1</quantity>
-		<description>Prodd Pellet</description>
-		<legality_class>4</legality_class>
-		<value>0.1</value>
-		<weight>0.06 lb</weight>
-		<reference>DFA104</reference>
-		<categories>
-			<category>Ammunition</category>
+			<category>UsesAmmoType:Prodd</category>
```
Comment (richardwilkes) [inline Library/Equipment/Dungeon Fantasy RPG.eqp]: Why did you remove the Prodd Pellet?

File: Library/Advantages/GURPS Lite Advantages.adq [hunk 2]
```diff
@@ -0,0 +1,1582 @@
+<?xml version="1.0" encoding="US-ASCII" ?>
```
Comment (richardwilkes) [inline Library/Advantages/GURPS Lite Advantages.adq]: This file should be renamed to take the "Advantages" part out of it, since that should be obvious given that it is within the Advantages folder.

File: Library/Equipment/GURPS Lite Equipment.eqp [hunk 3]
```diff
@@ -0,0 +1,3453 @@
+<?xml version="1.0" encoding="US-ASCII" ?>
```
Comment (richardwilkes) [inline Library/Equipment/GURPS Lite Equipment.eqp]: As with the advantages, the word "Equipment" should be removed from the file's name

File: Library/Output Templates/fantasy_grounds_npc_template.xml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      <p>@GENDER, Eyes: @EYES, Hair: @HAIR, Skin: @SKIN</p>
+      @NOTES_LOOP_START@@NOTE_FORMATTED@@NOTES_LOOP_END
+    </notes>
+    <pts type="number">@TOTAL_POINTS</pts>
+    <traits>
+      <sizemodifier type="string">@SIZE</sizemodifier>
+      <reactionmodifiers type="string"></reactionmodifiers>
+      <description type="string">
+        @ADVANTAGES_LOOP_START
+        @DESCRIPTION_PRIMARY (@POINTS)\r@ADVANTAGES_LOOP_END
```
Comment (richardwilkes) [inline Library/Output Templates/fantasy_grounds_npc_template.xml]: What is `\r` supposed to be doing?

File: Library/Output Templates/fantasy_grounds_pc_template.xml [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+        <id-@ID>
+          <name type="string">@DESCRIPTION_PRIMARY</name>
+          <points type="number">@POINTS</points>
+          <text type="string">@DESCRIPTION_MODIFIER_NOTES @DESCRIPTION_NOTES_PAREN @REF</text>
+        </id-@ID>@DISADVANTAGES_ALL_LOOP_END
+      </disadslist>
+
+      <culturalfamiliaritylist>@CULTURAL_FAMILIARITIES_LOOP_START
+        <id-@ID>
+          <name type = "string">@DESCRIPTION_PRIMARY</name>
```
Comment (richardwilkes) [inline Library/Output Templates/fantasy_grounds_pc_template.xml]: For consistency, remove the spaces on either side of the = sign here and elsewhere

File: Library/Skills/GURPS Lite Skills.skl [hunk 6]
```diff
@@ -0,0 +1,1460 @@
+<?xml version="1.0" encoding="US-ASCII" ?>
```
Comment (richardwilkes) [inline Library/Skills/GURPS Lite Skills.skl]: Please remove the word "Skills" from the file name

File: Library/Equipment/Dungeon Fantasy RPG.eqp (hunk 1 above)
Comment (crnormand) [reply Library/Equipment/Dungeon Fantasy RPG.eqp]: It is a duplicate of "Lead Pellet (Prodd)".

File: Library/Advantages/GURPS Lite Advantages.adq (hunk 2 above)
Comment (crnormand) [reply Library/Advantages/GURPS Lite Advantages.adq]: Done

File: Library/Equipment/Dungeon Fantasy RPG.eqp (hunk 1 above)
Comment (richardwilkes) [reply Library/Equipment/Dungeon Fantasy RPG.eqp]: Indeed! Good catch.

File: Library/Output Templates/fantasy_grounds_npc_template.xml (hunk 4 above)
Comment (crnormand) [reply Library/Output Templates/fantasy_grounds_npc_template.xml]: Fantasy Grounds converts "\r" to a newline when displaying the advantages.   The NPC format does not explicitly identify each advantage, but instead displays them in a semi-formatted text field.

File: Library/Output Templates/fantasy_grounds_pc_template.xml (hunk 5 above)
Comment (crnormand) [reply Library/Output Templates/fantasy_grounds_pc_template.xml]: Done.   Also, updated to include description notes/modifier notes and page references for all items that support them.

Comment (richardwilkes) [reply Library/Output Templates/fantasy_grounds_pc_template.xml]: Still seeing several things like `<name type = "string">` that should be `<name type="string">` for consistency.

Comment (crnormand) [reply Library/Output Templates/fantasy_grounds_pc_template.xml]: Got it that time.   I didn't know I needed the "-a" option.
